specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -849,6 +804,10 @@
             "mixed",
             "negative"
           ],
+          "x-ms-enum": {
+            "name": "TokenSentimentValue",
+            "modelAsString": false
+          },
```

Comment:
why was this added? cc @iscai-msft

Reply:
This changes the name of the generated enum class, and ensures that the enum is non-extensible, I'm assuming those were the reasons it was added, @laramume for confirmation

Reply:
So it is an implementation of the new guidelines defined here: https://github.com/microsoft/api-guidelines/pull/213. Thanks @iscai-msft.

Reply:
I'm actually not sure myself what this is. @laramume - please remove this if not needed.